client/packages/requisitions/src/ResponseRequisition/DetailView/ResponseLineEdit/ResponseLineEdit.tsx

```diff
@@ -326,7 +332,9 @@ export const ResponseLineEdit = ({
             Input={
               <NumericTextInput
                 width={INPUT_WIDTH}
-                value={draft?.remainingQuantityToSupply}
+                value={
+                  (draft?.supplyQuantity ?? 0) - (draft?.alreadyIssued ?? 0)
```

Comment:
![Screenshot 2024-11-27 at 9 02 54 AM](https://github.com/user-attachments/assets/3071c905-d178-4b45-b11a-eab50dd4503f)

If you issue more than supply quan in the outbound shipment then this shows as negative... I guess that's fine?? Maybe should be 0 idk